src/native/libs/System.Globalization.Native/build-nativeaot-shim-standalone.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+#
+
+# This script is used only for building libSystem.Globalization.Native.a
+# in the end-user's computer for NativeAOT purposes (static linking).
+# It is not used during the dotnet runtime build.
+
+# Currently, only Linux is supported
+
+SHIM_SOURCE_DIR=$1/source_code
+INTERMEDIATE_OUTPUT_PATH=$2
+
+if [ -d "$SHIM_SOURCE_DIR" ]; then
+    cp -r "$SHIM_SOURCE_DIR" "$INTERMEDIATE_OUTPUT_PATH"
```

Comment:
Why do you need to copy all sources over? Can you just point the build at the sources in the package and only set the output the intermediate path?